README.md

```diff
@@ -0,0 +1,81 @@
+# Conduit Connector ActiveMQ Classic
+
+The ActiveMQ Classic connector is one of [Conduit](https://conduit.io) plugins. The connector provides both a source and a destination connector for [ActiveMQ Classic](https://activemq.apache.org/components/classic/).
+
+It uses the [stomp protocol](https://stomp.github.io/) to connect to ActiveMQ.
```

Comment:
I like it that you're mentioning this. :+1: IMO, it would also be interesting for users to read what's actually in the records that are being read from AMQ (payload = message body, key = message ID, all metadata is preserved etc.). We should also describe the destination's behavior (the whole OpenCDC payload is written, content type etc.).